### PR TITLE
Feat/improve documentation

### PR DESCRIPTION
**Pull request by ewatch**
Hi,

I change some text paragraphs, added a screenshot to show how translations are displayed and 
made an additional internally in the documentation to finish the TODOs that were marked in the documentation.

Hope this helps and you like the improvements.

Best regards,
Jan

### CODE REVIEW

File: docs/guide/README.md [hunk 1]
```diff
@@ -2,14 +2,21 @@
...
 
 Each instance of an `entity` is called... an `instance`.
 
 Each `entity` in Sharp can be displayed:
-- in an `Entity List`, which is the list of all the `instances` for this `entity`: with some configuration and code, the user can sort the data, add filters, and perform a search. From there we also gain access to applicative `commands` applied to an `instance` or the whole list, and to a simple `state` changer (the publish state of an Article, for instance). All of that is described below.
-- In a `Show`, optionally, to display an `instance` details.
-- And in a `Form`, either to update or create a new `instance`.
+- in an `Entity List`, which is the list of all the `instances` for this `entity` (with some configuration and code, the user can sort the data, add filters and perform a search. The `Entity List` also provides the possibilty tp provide applicative `commands` to be executed on an `instance`or the whole `Entity List` and additionally provides a simple `state` changer which can be used to change the publish state in case of an Article `entity`. All of that is described below.
```
Comment (dvlpp) [inline docs/guide/README.md:17]: Mmm, I think this paragraph is not better than the existing one (which can be improved, I totally agree): there are repetitions (provide), opening "(" without closing, and I think a confusing presentation of the state handler, which seems to imply a "publish state"—that's only some se case example.

File: docs/guide/show-fields/embedded-entity-list.md [hunk 2]
```diff
@@ -84,4 +84,4 @@ There is no transformer, since Sharp will NOT look for an attribute in the insta
 
 ## A note on extending existing EntityList
 
-You can of course build a dedicated EntityList, but in many cases it's useful to extend one already needed in the main menu. In our stupid spaceship / pilots example, for instance, we can have a full pilots EntityList, presented in the main menu, with all pilots and some general commands, and another one for the spaceship Show, which extends the first one and maybe redefine its columns layout, or allow reorder because it had sense in this case. This is what's been done in [Sharp's Saturn demo project](https://sharp.code16.fr/sharp/).
\ No newline at end of file
+You can of course build a dedicated EntityList, but in many cases it's useful to extend one already needed in the main menu. In our demo project "Saturn" with the spaceships and pilots, for instance, we can have a full pilots EntityList, presented in the main menu, with all pilots and some general commands, and another one for the spaceship Show, which extends the first one and maybe redefine its columns layout, or allow reorder because it had sense in this case. This is what's been done in [Sharp's Saturn demo project](https://sharp.code16.fr/sharp/).
```
Comment (dvlpp) [inline docs/guide/show-fields/embedded-entity-list.md:87]: You are totally right on this one 🙄

**Review by dvlpp — CHANGES_REQUESTED** [review]
Hi @ewatch, thanks a lot for this! I think Sharp docs should be improved in many areas. I have a few remarks on your version, listed in the review.

File: docs/guide/README.md (hunk 1 above)
Comment (ewatch) [reply docs/guide/README.md]: Hi dvlpp,
fine for me. I tried to use shorter sentences and different words.
In your version there was still a typo that I fixed `tp`=> `to`.

Comment (dvlpp) [reply docs/guide/README.md]: Sorry, I wasn't clear: I did not provide a new version, it's your first one on which I just posted remarks. Your new version fixed a typo (there is another one on possibility, a "i" is missing), but my remarks stands...

Comment (ewatch) [reply docs/guide/README.md]: I think I was a little bit confused.
I hope the PR is now in the proper state else I will create a whole new PR.
I understood that you want to have the original paragraph back so I made a new commit for this.
Please check if this is fine now.

**Pull request closed without merging**